package.json

```diff
@@ -22,8 +22,7 @@
 		"@angular/router": "~12.2.9",
 		"@datx/core": "^2.2.3",
 		"@datx/jsonapi": "^2.2.3",
-		"@datx/jsonapi-angular": "^2.2.3",
-		"mobx": "^6.3.3",
+		"@datx/jsonapi-angular": "file:datx-jsonapi-angular-next.tgz",
```

Comment:
Instead of pointing to a file, it might be better to point to a repository branch with:
```
npm install "https://github.com/infinum/datx.git#branch-name" --save
```
In any case I would not merge this until datx stable contains these changes.

Reply:
I did this only so that I can test that it works before publishing. I will publish a new version of `@datx/jsonapi-angular` and update the dependency for the example app.

Reply:
TIL: this type of linking that you suggested does not work because we have a monorepo, so the imports do not work correctly:

```[0] Error: src/app/services/artists/artists.service.ts:2:35 - error TS2307: Cannot find module '@datx/jsonapi-angular' or its corresponding type declarations.
[0] 
[0] 2 import { CollectionService } from '@datx/jsonapi-angular';
```

This type of link works with source code, but we need the npm artifact with built files.

Anyway, I will update the example after the new version of datx is published.